spec.html

```diff
@@ -438,11 +438,12 @@ <h1>%PrivateName% ( )</h1>
       </emu-clause>
 
       <emu-clause id="sec-private-name-object" aoid=PrivateNameObject>
-        <h1>PrivateNameObject ( _name_ )</h1>
+        <h1>PrivateNameObject ( _name_, _restricted_ )</h1>
         <p>When PrivateNameObject is called with Private Name _name_, the following steps are taken:</p>
```

Comment:
`"and Boolean _restricted_"`? (and then should there be an Assert that _restricted_ is a boolean, or is that implied?)